### PR TITLE
Swedish translations: Fix missing translations

### PR DESCRIPTION
**Pull request by antonborgstrom**
## What does this PR do?

- Adds missing locale strings in Swedish in ftl format
- Modifies some existing incorrect strings in Swedish

## These changes will impact:

- [x] commenters
- [x] moderators
- [x] admins
- [ ] developers

### CODE REVIEW

File: locales/sv/stream.ftl [hunk 1]
```diff
@@ -24,45 +45,140 @@ comment-count-text =
...
+comments-permalinkView-reportIllegalContent-additionalComments-validCommentURLError = Detta är inte en giltig URL. Vänligen ange en giltig kommentars-URL
+comments-permalinkView-reportIllegalContent-additionalComments-uniqueCommentURLError = Du har redan lagt till denna kommentar i rapporten. Vänligen lägg till en unik kommentars-URL
+omments-permalinkView-reportIllegalContent-additionalComments-validCommentURLLengthError = Längden på ytterligare kommentars-URL överskrider maximalt.
+comments-permalinkView-reportIllegalContent-additionalComments-previouslyReportedCommentError = Du har tidigare anmält denna kommentar för att innehålla olagligt innehåll. Du kan bara anmäla en kommentar av denna anledning en gång.
+comments-permalinkView-reportIllegalContent-confirmation-successHeader = Vi har mottagit din anmälan om olagligt innehåll
+comments-permalinkView-reportIllegalContent-confirmation-description = Your report will now be reviewed by our moderation team. You will receive a notification once a decision is made. If the content is
```
Comment (kabeaty) [inline locales/sv/stream.ftl]: I noticed that this translation is still in English. Did you want to translate it?

**Review by kabeaty — APPROVED** [review]
Thank you for submitting these translations! They look good. I did leave one comment where I noticed a string was still not translated.

File: locales/sv/stream.ftl (hunk 1 above)
Comment (losowsky) [reply locales/sv/stream.ftl]: @antonborgstrom

Comment (antonborgstrom) [reply locales/sv/stream.ftl]: Thanks for the review @losowsky and @kabeaty! 
You are right, that string was missed. I pushed additional commit to resolve the translation now.

Comment (kabeaty) [reply locales/sv/stream.ftl]: Thank you!

**Pull request merged**